cassandra/README.md

```diff
@@ -53,47 +51,71 @@ See the [sample  cassandra.d/conf.yaml][7] for all available configuration optio
     ```
 
     Change the `path` and `service` parameter values and configure them for your environment.
-    See the [sample  cassandra.d/conf.yaml][7] for all available configuration options.
+    See the [sample  cassandra.d/conf.yaml][5] for all available configuration options.
+
+    To make sure that stacktraces are properly aggregated as one single log, a [multiline processing rule][7] can be added.
+
+3. [Restart the Agent][6].
+
+#### Containerized
+
+For containerized environments, see the [Autodiscovery Integration Templates][8] for guidance on applying the parameters below.
+
+##### Metric collection
 
-    To make sure that stacktraces are properly aggregated as one single log, a [multiline processing rule][8] can be added.
+For containerized environments, see the [Autodiscovery with JMX][9] guide.
```

Comment:
IMHO I doubt that users will find this useful. Can we provide more specific information here to set up the AD integration for Cassandra? The JMX guide does not provide concrete examples for Cassandra either and leaves the user with a complicated understanding of many different pieces.

Reply:
The guide is focused on Kafka but the setup process is the same for all JMX integration. 
we won't create a dedicated guide for all specific JMX integrations unless there is a real divergence between them.

Reply:
Ideally, we should create a template of JMX AD setup instructions and display it in-app with the corresponding value for a given integration.